hs_modflow_modelinstance/tests/test_modflow_modelinstance_metadata.py

```diff
@@ -902,70 +902,18 @@ def test_public_or_discoverable(self):
                                             extract_metadata=False)
         utils.resource_file_add_process(resource=self.res, files=files, user=self.user,
                                         extract_metadata=False)
-        self.assertFalse(self.res.has_required_content_files())
-        self.assertFalse(self.res.can_be_public_or_discoverable)
-
-        # add .nam file
-        files = [UploadedFile(file=self.sample_nam_obj, name=self.sample_nam_obj.name)]
-        utils.resource_file_add_pre_process(resource=self.res, files=files, user=self.user,
-                                            extract_metadata=False)
-        utils.resource_file_add_process(resource=self.res, files=files, user=self.user,
-                                        extract_metadata=False)
-        self.assertFalse(self.res.has_required_content_files())
-        self.assertFalse(self.res.can_be_public_or_discoverable)
-
-        # add reqd files except 2
-        for f in self.file_list[2:]:

```

Comment:
Because we removed the restrictions to have these files uploaded before making the resource public or discoverable.